docs/tutorial.md

```diff
@@ -183,8 +183,8 @@ the previous contents of `edge_buf`, which is why we choose to discard them
 and use the `discard_write` access mode.
```

Comment:
This paragraph would need updating as well; I'm doing a general pass on the documentation right now where I'll address this.